### PR TITLE
multiple events example added

### PR DESCRIPTION
**Pull request by vitaly-zdanevich**

### CODE REVIEW

**Comment by AurelioDeRosa** [comment]
@kswedberg Can you check the wording before I merge this PR? Thank you.

File: entries/on.xml [hunk 1]
```diff
@@ -265,13 +265,21 @@ $( "body" ).on( "click", "p", function() {
...
-    <desc>Cancel a link's default action using the <code>.preventDefault()</code> method.</desc>
+    <desc>Cancel a link's default action using the <code>.preventDefault()</code> method:</desc>
     <code><![CDATA[
 $( "body" ).on( "click", "a", function( event ) {
   event.preventDefault();
 });
 ]]></code>
   </example>
+  <example>
+    <desc>Also you can attach multiple events - for example this is usefull if you want to fire you function when user enter the mouse on some element and move out:</desc>
```
Comment (kswedberg) [inline entries/on.xml]: I think this description can be simplified quite a bit…
Attach multiple events—one on `mouseenter` and one on `mouseleave` to the same element:

File: entries/on.xml [hunk 2]
```diff
@@ -265,13 +265,21 @@ $( "body" ).on( "click", "p", function() {
...
+    <code><![CDATA[
+$( "#cart" ).on( "mouseenter mouseleave", function( event ) {
+  $('#cart').toggleClass('active');
```
Comment (kswedberg) [inline entries/on.xml]: Why not use `$( this )` here? Also, should have spaces around the quotes, and the quotes should be double, not single.

File: entries/on.xml [hunk 3]
```diff
@@ -265,13 +265,21 @@ $( "body" ).on( "click", "p", function() {
...
 $( "body" ).on( "click", "a", function( event ) {
   event.preventDefault();
 });
 ]]></code>
   </example>
+  <example>
+    <desc>Attach multiple events—one on mouseenter and one on mouseleave to the same element:</desc>
+    <code><![CDATA[
+$( "#cart" ).on( "mouseenter mouseleave", function( event ) {
+  $( $this ).toggleClass("active");
```
Comment (AurelioDeRosa) [inline entries/on.xml]:
```js
$( $this ).toggleClass("active");
```

should be 

```js
$( this ).toggleClass( "active" );
```

**Comment by AurelioDeRosa** [comment]
PR merged, thank you for the updates.

**Pull request closed without merging**